tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Skew.cpp

```diff
@@ -113,11 +115,74 @@ void test(const gsl::not_null<Generator*> generator) {
   CHECK(skew_map == skew_map_deserialized);
   CHECK_FALSE(skew_map != skew_map_deserialized);
 }
+
+void test_specific_points() {
+  const std::string function_of_time_name{"Skew"};
+  const double time = 0.0;
+  std::unordered_map<std::string, FoftPtr> f_of_t_list{};
+  // Use pi/4 so math is easy
+  f_of_t_list[function_of_time_name] = std::make_unique<Polynomial>(
+      time,
+      std::array{DataVector{M_PI_4, M_PI_4}, DataVector{2, 0.0},
+                 DataVector{2, 0.0}},
+      std::numeric_limits<double>::infinity());
+
+  const std::array<double, 3> center{-1.0, 1.0, 0.0};
+  const double outer_radius = 100.0;
+
+  const CoordinateMaps::TimeDependent::Skew skew_map{function_of_time_name,
+                                                     center, outer_radius};
+
+  std::array<double, 3> test_point{};
+  {
+    INFO("Center");
+    test_point = center;
+    CAPTURE(test_point);
+
+    const auto mapped_point = skew_map(test_point, time, f_of_t_list);
+    CHECK_ITERABLE_APPROX(mapped_point, (std::array{0.0, 1.0, 0.0}));
+    const auto inverse_point =
+        skew_map.inverse(mapped_point, time, f_of_t_list);
+    CHECK(inverse_point.has_value());
+    CHECK_ITERABLE_APPROX(inverse_point.value(), test_point);
+    // Jacobian not defined at the center
```

Comment:
do we not need it?

Reply:
Unclear. SpEC didn't handle this specially (and thus would get a division by zero error if the center was passed to the jacobian).